playbooks/infra-prometheus/inventory/group_vars/prometheus_scraper.yml

```diff
@@ -0,0 +1,13 @@
+---
+smtp_host: my_smtp_host
+smtp_port: my_smtp_port
+from: my_smtp_from
+smtp_username: 'smtp_user'
+smtp_password: 'smtp_password'
+smtp_tls: 'false'
+
+
+datasources:
+- name: "openshift-1"
+  datasource_url: "https://prometheus-k8s-openshift-monitoring.apps.openshift-1.example.com"
+  bearer_token: "prometheus-k8s-secret-token"
```

Comment:
This file has merge conflicts.
Also, did you intend to leave the example datasource be here?

Reply:
I will move the example datasource to readme if you think it's better, but I would like to keep it in this file as well as inline comments at least. The prometheus_scraper file is going to contain more variables for those exporters running locally on the prometheus node (e.g. targets for ssl_exporter).

Reply:
The merge conflict has been resolved.